src/serial/serialcommands.cpp

```diff
@@ -153,6 +172,32 @@ namespace SerialCommands {
                 logger.info("[TEST] Sensor[0] sent some data, looks working.");
             }
         }
+
+        if (parser->equalCmdParam(1, "WIFISCAN")) {
+			logger.info("[WSCAN] Scanning for WiFi networks...");
+
+			// Scan would fail if connecting, stop connecting before scan
+			if (WiFi.status() != WL_CONNECTED) WiFi.disconnect();
+
+			WiFi.scanNetworks();
```

Comment:
I did try it on ESP8266:
It did not work for me after a short time after boot, after the line
`[INFO ] [WiFiProvisioning] SmartConfig started` it did return everytime:
`[INFO ] [SerialCommands] [WSCAN] Scan failed!`

i would also love to have this function async, because it will probably mess up some IMUs because they need to fetch data or they may timeout.